Include/pymath.h

```diff
@@ -227,4 +227,10 @@ PyAPI_FUNC(void) _Py_set_387controlword(unsigned short);
  * behavior. */
 #define _Py_InIntegralTypeRange(type, v) (_Py_IntegralTypeMin(type) <= v && v <= _Py_IntegralTypeMax(type))
 
+/* Return the smallest integer k such that n < 2**k, or 0 if n == 0.
+ * Equivalent to floor(lg(x))+1.  Also equivalent to: bitwidth_of_type -
+ * count_leading_zero_bits(x)
+ */
+extern unsigned int _Py_bit_length(unsigned long d);
```

Comment:
You should use PyAPI_FUNC(unsigned int) rather than extern.

By the way, _Py symbols should be moved to a new Include/cython/pymath.h header IMHO, and excluded from the stable API.

Reply:
(1) Done.

(2) Sounds reasonable. Would you prefer to start this new file now (and move the other private functions later), or a seperate patch that moves all of the existing private functions including this new one?